gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -258,7 +258,7 @@ class {{ service.name }}GrpcTransport({{ service.name }}Transport):
             )
 
         self._grpc_channel = self._base_channel
-        if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(std_logging.DEBUG):  # pragma: NO COVER
+        if CLIENT_LOGGING_SUPPORTED and not _LOGGER.isEnabledFor(std_logging.NOTSET):  # pragma: NO COVER
```

Comment:
We only set up these channels when we set up the transport, right, and they're used for various RPC calls?

Then we will run into a problem if the user changes their logging config at run time. Specifically:
1. The user sets things up and does their first RPC call with no logging.
2. The user code then sets up logging either for this module or one of its ancestors.
3. The user makes an RPC call, expecting it to be logged, but since the transport was set up without logging, this interceptor never gets called.

I think we should always add the logging interceptor, and have it do the check when it is called. If logging is off, it's a single check and a pass-through. And if the logging config gets changes in between RPC calls, it will pick it up. This means removing the logging check here, so my other comment about changing it here does not apply.